Content.Server/Nutrition/EntitySystems/SliceableFoodSystem.cs

```diff
@@ -34,6 +34,9 @@ private void OnInteractUsing(Entity<SliceableFoodComponent> entity, ref Interact
         if (args.Handled)
             return;
 
+        if (!TryComp<UtensilComponent>(args.Used, out var utensil) || (utensil.Types & UtensilType.Knife) == 0)
+            return;
+
```

Comment:
Doesn't consider the food solution being missing when you copied it.

Would be better to just make a new method for it.

Reply:
Makes sense! Let me know if this looks good